src/sentry/api/serializers/models/organization_member/base.py

```diff
@@ -67,8 +67,9 @@ def serialize(
             "email": obj.get_email(),
             "name": obj.user.get_display_name() if obj.user else obj.get_email(),
             "user": attrs["user"],
-            "role": obj.role,
-            "roleName": roles.get(obj.role).name,
+            "role": obj.role,  # Deprecated, use orgRole instead
```

Comment:
out of curiosity, why are deprecated fields not being removed?

Reply:
1. Removing the keys will break compatibility to the frontend. I need to make sure the entire frontend uses the new keys before I can remove the old keys.
2. This is a public endpoint and it will be a breaking change to the API, and that's out of scope to my project.

Reply:
Can you create a jira ticket too so we remember to remove the old keys?